packages/formstr-app/src/components/v1/QuestionCard/CardHeader.tsx

```diff
@@ -0,0 +1,22 @@
+import { ArrowDownOutlined, ArrowUpOutlined } from "@ant-design/icons";
+import { ReactComponent as Asterisk } from "../../../Images/asterisk.svg";
+
+const CardHeader = () => {
+  return (
+    <div style={{ display: "flex", paddingBottom: 10 }}>
+      <div style={{ padding: 5, paddingLeft: 0 }}>
+        <ArrowUpOutlined style={{ fontSize: 12 }} />
+      </div>
+      <div style={{ padding: 5 }}>
+        <ArrowDownOutlined style={{ fontSize: 12 }} />
```

Comment:
we can use `rem` for font size. It is much more controllable, configurable and helpful for some people(accessible).

Some people increase the general font size of the browser at the body level. rem is basically a unit which is proportional to the font size of the body. So if we increase the font size of the body, all other components font size would scale correspondingly